documentation/configure.markdown

```diff
@@ -387,6 +387,18 @@ runs every `gc-interval` minutes.
 
 If unset, the default value is 14 days.
 
+### `resource-events-ttl`
```

Comment:
Our other ttls are singular, `node-ttl` and `report-ttl` should we maintain that for this one as well?

Reply:
I'm not sure this is important.

Reply:
Hmm, that sounds plausible.